src/models/models.ts

```diff
@@ -0,0 +1,64 @@
+export interface TimerState {
+  time: TimerCount; 
```

Comment:
Typescript syntax is to use `:` after a value to denote its type. So `let asd: number` defines asd as type number. Type can also be automatically inferrered without being explicitly specified, like for example if I instead did `let asd = 5`, typescript is smart enough to infer that asd is of type number